test/test_linalg.py

```diff
@@ -6401,6 +6405,62 @@ def int4_mm(a, b_int32, b_scales_and_zeros):
         mean_err = ((res - ref).abs() / ref).mean()
         self.assertTrue(mean_err < 0.05)
 
+    @skipIfNoKleidiAI
+    @onlyCPU
+    @parametrize("m", [1, 32])
+    @parametrize("k", [128, 256])
```

Comment:
given these quantized kernels are more relevant for LLMs, please add couple of tests with those model sizes as well, may be for 1B and 3B parameter models

Reply:
Our kernels are not for LLMs. They are general quantized matmul kernels. I reviewed the test cases and kept the dimensions same as other test cases. Just to accomodate your request I will add larger dimensions.